demos/serverless/README.md

```diff
@@ -28,6 +28,19 @@ npm install
 npm run deploy -- -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
 ```
 
+#### ChimeSDKMeetings Namespace vs Chime NameSpace
+The Chime and the ChimeSDKMeetings are both Amazon Chime's AWS clients to help builders create the Amazon Chime SDK meeting, attendees, enable transcription features and so on. Going forward, we will be using ChimeSDKMeetings client as the default client.
+
+For local deployment, you can provide an additional parameter -u false if you want to use the legacy Chime SDK client over the ChimeSDKMeetings client.
```

Comment:
Is this just for our internal testing? Can we make that clear to builders here?

Reply:
@hensmi-amazon I am not sure if we should provide this information as just being used for internal testing as we want ChimeSDKMeetings to be used as default client and only if some builders spcifically wants to use chime client they can provide the -u false flag.

Reply:
Why would a builder want to use the legacy client?  If theres a reason we should put that in documentation as well.

Reply:
I could not think of any particular reason so i added the description for new -u command line parameter and that it is primarly intended for internal testing